app/src/main/java/ch/sdp/vibester/activity/AuthenticationActivity.kt

```diff
@@ -15,16 +15,20 @@ import com.google.android.gms.auth.api.signin.GoogleSignInClient
 import com.google.android.gms.auth.api.signin.GoogleSignInOptions
 import com.google.android.gms.tasks.Task
 import com.google.firebase.auth.AuthResult
+import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
 
+
```

Comment:
It will be great if you can add some documentation to this class and its non-trivial functions